tests/InitCommandTest.php

```diff
@@ -926,4 +930,50 @@ public function testRunWithClerkAdditionalAdminsWithoutDefaultAdmin(): void
             ->expectsConfirmation('Do you want to uninstall project-initializator package?')
             ->assertExitCode(0);
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testDefaultAdminsCredentials()
+    {
+        $this->mockNativeFunction(
+            'RonasIT\ProjectInitializator\Commands',
+            $this->mockAdminDefaultPassword('123456'),
+            $this->callFilePutContent('database/migrations/2018_11_11_111111_add_default_admin.php', $this->getFixture('users_default_admin_default_credentials.php')),
```

Comment:
could we use already exited migrations? or these fixtures has some specific data?